src/components/MiniDrawer.js

```diff
@@ -141,13 +141,13 @@ class MiniDrawer extends React.Component {
           </section>
 
           <Divider />
-          <list>
+          {/* <ul> */}
```

Comment:
I think this may need to be <List> but idk. And idk why. I'll retry to remmber

Reply:
ok! I didn't think it was necessary. We can put it back!